frontend/src/scenes/notebooks/NotebooksTable/NotebooksTable.tsx

```diff
@@ -45,12 +48,24 @@ export function NotebooksTable(): JSX.Element {
         useValues(notebooksTableLogic)
     const { loadNotebooks, setFilters, tableSortingChanged } = useActions(notebooksTableLogic)
     const { selectNotebook } = useActions(notebookPanelLogic)
+    const { hasAvailableFeature } = useValues(userLogic)
 
     useOnMountEffect(loadNotebooks)
 
     const columns: LemonTableColumns<NotebookListItemType> = [
         titleColumn() as LemonTableColumn<NotebookListItemType, keyof NotebookListItemType | undefined>,
 
+        ...(hasAvailableFeature(AvailableFeature.TAGGING)
+            ? [
+                  {
+                      title: 'Tags',
```

Comment:
As discussed, let's not have a tags column, but rather show a pill like the [TEMPLATE] one.